README.md

```diff
@@ -55,79 +43,37 @@ namespace loc {
 }
 ```
 
-Generate a C++ header `locations.hpp` from it.
-```shell
-./generator/app.py --gen cpp --schema locations.flatdata --output-file locations.hpp
-```
-
-Serialize some data:
-```cpp
-// Compile with: c++ -std=c++11 writer.cpp -Iflatdata-cpp/include -Lbuild/flatdata-cpp -lflatdata -lboost_system -lboost_filesystem -o writer
-#include "locations.hpp"
-int main() {
-  auto storage = flatdata::FileResourceStorage::create("locations.archive");  // create storage
-  auto builder = loc::LocationsBuilder::open(std::move(storage));             // create builder
-  auto pois = builder.start_pois();
-
-  uint32_t x, y;
-  while(std::cin >> x >> y) {
-      loc::PointMutator poi = pois.grow();
-      poi.x = x;
-      poi.y = y;
-  }
-  pois.close();  // flush not yet written data to disk
-}
-```
-
-And finally, read the serialized data:
-
-```cpp
-// Compile with: c++ -std=c++11 reader.cpp -Iflatdata-cpp/include -Lbuild/flatdata-cpp -lflatdata -lboost_system -lboost_filesystem -o reader
-#include "locations.hpp"
-#include <iostream>
-int main() {
-    auto storage = flatdata::FileResourceStorage::create("locations.archive");  // open storage
-    auto archive = loc::Locations::open(std::move(storage));              // create archive
-    for (loc::Point point : archive.pois()) {                             // iterate through pois
-        std::cout << point.to_string() << std::endl;
-    }
-    return 0;
-}
-```
+### Generating a module
 
-For more examples cf. the [examples](examples) directory.
+Flatdata relies on a generator that takes a `.flatdata` file as an input and
+generates a module for one of the supported languages.
 
-## Library Layout
+The following languages are supported:
 
-Library is organized as follows:
+  * First-class citizen implementations:
+    * **[C++](./flatdata-cpp)** - used extensively, tested excessively normally receives features first
+    * **[Rust](./flatdata-rs)** - the newest addition to the family
+  * Read-only implementations:
+    * **[Python](./flatdata-py)** - used mostly for debugging purposes
```

Comment:
I would say python is rather for `inspecting the data`. Not if it is more to the point. Feel free not to change.